app/jobs/reports/verification_errors_report.rb

```diff
@@ -0,0 +1,62 @@
+require 'identity/hostdata'
+require 'csv'
+
+module Reports
+  class VerificationErrorsReport < BaseReport
+    REPORT_NAME = 'verification-errors-report'.freeze
+
+    include GoodJob::ActiveJobExtensions::Concurrency
+
+    good_job_control_concurrency_with(
+      total_limit: 1,
+      key: -> { "#{REPORT_NAME}-#{arguments.first}" },
+    )
+
+    def perform(_date)
+      csv_reports = []
+      configs = IdentityConfig.store.verification_errors_report_configs
+      configs.each do |report_hash|
+        issuers = report_hash['issuers']
+        report = verification_errors_data_for_issuers(issuers)
+        csv_reports << report
+      end
+      csv_reports
+    end
+
+    private
+
+    def verification_errors_data_for_issuers(issuers)
+      csv = CSV.new('', row_sep: "\r\n")
+      csv << %w[uuid welcome_view_at error_code]
+      issuers.each do |issuer|
+        transaction_with_timeout do
+          rows = ::VerificationErrorsReport.call(issuer, 24.hours.ago)
```

Comment:
let's use `_date` here so we could backfill data in the future if needed

maybe turn it into into a range `(date.beginning_of_day - 1).beginning_of_day..date.beginning_of_day`